curriculum/dfe/content.json

```diff
@@ -24,6 +28,16 @@
       {"id": "Table", "title": "Table", "isTileTool": true},
       {"id": "Drawing", "title": "Drawing", "isTileTool": true},
       {"id": "Dataflow", "title": "Dataflow", "isTileTool": true},
+      {
+        "id": "Simulator",
+        "title": "Simulator",
+        "isTileTool": true
+      },
```

Comment:
Did you want to start this way with the Simulator on the toolbar? That is fine if so, just so we realize the ultimate goal is to show in the curriculum but not in the toolbar.

Reply:
Yeah I think it's good to have a button that works, even if we don't use it in any live units. I've got a copy on the left in this curriculum so you can add a simulator tile in either way 👍